src/frontend/Komponenter/Personoversikt/PersonOversiktSide.tsx

```diff
@@ -172,7 +175,7 @@ const PersonOversikt: React.FC<Props> = ({
                 <Tabs
                     value={path}
                     onChange={(fane) => {
-                        navigate(`/person/${fagsakPerson.id}/${fane}`);
+                        navigate(`/person/${fagsakPersonId}/${fane}`);
```

Comment:
Vi må bruke fagsakPersonId fra urlen - ikke den eksisterende staten.
Før oppgraderingen av react-router-dom stod det ikke `/person/${fagsakPersonId/` i `navigate`-funksjonen, men denne måtte med for at redirecten skulle fungere som forventet.